packages/stryker-vue-mutator/test/integration/VueMutator.it.ts

```diff
@@ -0,0 +1,87 @@
+import { MutatorFactory } from 'stryker-api/mutant';
+import VueMutator from '../../src/VueMutator';
+import { Config } from 'stryker-api/config';
+import { File } from 'stryker-api/core';
+import { expect } from 'chai';
+require('../../src/index');
+require('stryker-javascript-mutator');
+require('stryker-typescript');
+
+describe('VueMutator', () => {
+  describe('JavaScript project', () => {
+    it('should generate mutants', () => {
+      sandbox.stub(MutatorFactory.instance(), 'knownNames').returns(['javascript']);
+      const vueCode = `<template>
+      <span id="msg">{{ message }}</span>
+      </template>
+      
+      <script>
+      export default {
+        data () {
+          return {
+            message: 'hello!'
+          }
+        },
+        doMath () {
+          return 1 - 2;
+        }
+      }
+      </script>
+      
+      <msg>
+      #app {
+      text-align: center;
+      }
+      </msg>`;
+      const jsCode = `"use strict";
+      var fs = require('fs');
+      function checkAge(user) {
+        return user.age >= 18;
+      }`;
+      const files = [new File('AppComponent.vue', vueCode), new File('age.js', jsCode)];
+      const sut = new VueMutator(new Config());
+
+      const mutants = sut.mutate(files);
+
+      expect(mutants.length).to.equal(8);
```

Comment:
This test is pretty flaky, right? Whenever we update our js mutator, this might fail?